### DISCUSSION TITLE
FileUpload reorder images and sync with spatie order_column

### DISCUSSION THREAD

**Discussion by creativeplus2** [filamentphp/filament#1470] [Ideas]
<img width="548" alt="Screen Shot 2022-02-06 at 16 05 00" src="https://user-images.githubusercontent.com/676423/152674245-bd79a8cf-2d64-446e-a11d-409139041de0.png">
FileUpload reorder draganddrop image and sync with spatie media library "order_column" so user can reorder the image after upload. I think filepond already can drag reorder in file-upload.blade.php  in input  add data-allow-reorder="true". 

Thanks

Chrostian

**Comment by danharrin** [comment]
This will be available in v2.10.0.
